detail: add image onerror inline js to script

jennur opened the pull request:
Closes https://github.com/inveniosoftware/invenio-app-rdm/issues/1616

- Adds `onerror` logic to function

kpsherva commented on invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html:
```diff
@@ -25,6 +25,15 @@
 {%- block page_body %}
+
+  <!-- Image placeholder -->
+  <script type="text/javascript">
```
could it be moved to .js file? for example theme.js ?

jennur commented:
## Updates

- To make this more reusable I added a `imagePlaceholder` function to `invenio-theme` so that it is available across all repos. 
- Added `.has-placeholder` to community-image, which is connected to the `imagePlaceholder`-function

I.e. this PR now needs: https://github.com/inveniosoftware/invenio-theme/pull/296
